### PR TITLE
colab validation gelindi

### PR DESCRIPTION
**Pull request by mertyigt**

### CODE REVIEW

File: ISNet/model/loss.py [hunk 1]
```diff
@@ -40,13 +40,15 @@ def update(self, val, n=1):
...
         self.avg = self.sum / self.count
+        
 class SoftIoULoss(nn.Module):
     def __init__(self):
         super(SoftIoULoss, self).__init__()
     def forward(self, preds, gt_masks):
         if isinstance(preds, list) or isinstance(preds, tuple):
             loss_total = 0
             for i in range(len(preds)):
+                print("MEEEEEEEEEEEEEEEEEEEEEEEERT")
```
Comment (MucahidRidvanKaplan) [inline ISNet/model/loss.py:51]: silinecek

**Comment by MucahidRidvanKaplan** [comment]
merge edildi.
